src/status/StatusPage.js

```diff
@@ -56,12 +56,8 @@ const StatusPage = ({
         <AskToEnable
           className='mt3'
           label={t('AskToEnable.label')}
-          yesLabel={t('app:actions.ok')}
-          noLabel={t('app:actions.noThanks')}
-          detailsLabel={t('app:actions.moreInfo')}
-          detailsLink='#/settings/analytics'
-          onYes={doEnableAnalytics}
-          onNo={doDisableAnalytics} />
+          yesLabel={t('app:actions.close')}
+          onYes={doEnableAnalytics} />
```

Comment:
I just want to confirm: metrics will have already been enabled prior to user clicking anything on the banner, yes?